pricing.js

```diff
@@ -7,30 +27,23 @@ module.exports.calculateProductPrice = function (product, employee, coverageLeve
       for (var i = 0; i < coverageLevels.length; i++) {
         var coverageAmount = coverageLevels[i].coverage
 
-        price += (coverageAmount / product.cost.costDivisor) * product.cost.price
+        price = lifePrice(coverageAmount, product, price)
       }
 
-      if (product.employerContribution.mode === 'dollar') {
-        price = price - product.employerContribution.contribution
-      } else {
-        dollarsOff = price * (product.employerContribution.contribution / 100)
-        price = price - dollarsOff
-      }
+      var lifeDiscount = discount(dollarsOff, product, price)
+
+
+      return formatPrice(lifeDiscount)
 
-      return parseInt(price * 100) / 100
     case 'ltd':
       var salaryPercentage = product.coveragePercentage / 100
```

Comment:
This line is also part of the process for calculating ltd price so really should live with the rest of that code in the ltdPrice function.